src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -10,23 +10,24 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 import frc.robot.Ports;
-import frc.robot.subsystems.LoggedSubsystem;
 import frc.robot.subsystems.intake.commands.HoldIntakeInPlace;
 import frc.robot.utils.units.UnitModel;
+import org.littletonrobotics.junction.Logger;
 
-public class Intake extends LoggedSubsystem<IntakeLoggedInputs> {
+public class Intake extends SubsystemBase {
     private static Intake INSTANCE;
     private final CANSparkMax motor = new CANSparkMax(Ports.Intake.INTAKE_MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless);
     private final TalonFX angleMotor = new TalonFX(Ports.Intake.ANGLE_MOTOR);
     private final UnitModel unitModel = new UnitModel(IntakeConstants.TICKS_PER_DEGREE);
-
+    private ControlMode angleMode;
```

Comment:
CTRE enums should not be used in a subsystem's class that isn't supposed to know the hardware powering it.